internal/brain/free_three_test.go

```diff
@@ -44,14 +44,15 @@ func TestCheckSequenceForF3(t *testing.T) {
 
 		// should find 3 overlapping
 		{[]int{0, 1, 1, 0, 1, 0, 1, 1, 0, 1, 0}, 1, []int{0, 3, 5}},
+
+		// Enemy inside
+		{[]int{0, 1, 1, 2, 1, 0}, 1, []int{}},
 	}
 
 	for _, table := range tables {
 		actual := CheckSequenceForF3(table.sequence, table.playerId)
 		if !reflect.DeepEqual(actual, table.expected) {
 			t.Errorf("⛔️ F3 in sequence %v: got %v, expected %v\n", table.sequence, actual, table.expected)
```

Comment:
This file should not be updated; new test case is already included (under "Mixed Players" comment in test case list) and I would like the log with ✅ for verbose mode!